### DISCUSSION TITLE
Save 1D variables to csv/excel file

### DISCUSSION THREAD

**Discussion by manjunathnilugal** [pybamm-team/PyBaMM#2255] [Q&A] [answered]
Dear team,

I am trying to save the results of  the 1D variables such as (**"Electrolyte concentration"**)  in csv file, but getting error as "1D variable cannot be saved"
Is there any document to save 1D variables wrt time, and space in csv file.

Regards,
Manjunath N

**Comment by valentinsulzer** [comment, marked as answer]
No, this is not implemented

**Reply by manjunathnilugal** [reply]
Oh, okay , 
Thanks for the response sir,

**Reply by valentinsulzer** [reply]
It would be pretty easy for you to implement this btw, if you want to open a pull request to do it that would be welcome

**Reply by manjunathnilugal** [reply]
Sir, I tried saving the data in the **matlab** format and this solution worked for me.
